src/main/java/de/tum/in/www1/artemis/config/Constants.java

```diff
@@ -235,6 +235,8 @@ public final class Constants {
 
     public static final String VOTE_EMOJI_ID = "heavy_plus_sign";
 
+    public static final int MAX_VALUE_PROGRAMMING_RESULT_INTS = 255; // Size of tinyInt in SQL, that is used to store these values
```

Comment:
```suggestion
    /**
     * Size of tinyInt in SQL, that is used to store these values.
     */
    public static final int MAX_VALUE_PROGRAMMING_RESULT_INTS = 255;
```
By changing it to a proper JavaDoc the information is shown on hover in IDEs.